package.json

```
@@ -115,12 +115,12 @@
   },
   "homepage": "https://renovatebot.com",
   "engines": {
-    "node": "^12.22.0 || >=14.15.0",
+    "node": "^14.15.0 || >=16.13.0",
     "yarn": "^1.17.0"
   },
   "engines-next": {
     "description": "Versions other than the below are deprecated and a warning will be logged",
-    "node": ">=14.15.0"
+    "node": "^14.15.0 || >=16.13.0"
```

```suggestion
    "node": "^14.15.0 || ^16.13.0"
```

Disallow node v17 for now (issue a warning)?

---

My instinct is to disallow node 17, but I have some recollection of getting ourselves in trouble with eager deprecations in the past. But maybe was that for node 12 I was thinking and not a new/non-LTS?

Our goal should be:
- Block any node which *won't work*
- Warn for any we don't support

I'm happy to block node 15 even if it probably works, particularly because there's no reason to use it. Is there any reason to use node 17?

---

not, yet. The idea above is to only issue a warning, if somebody uses node v17. As i think there are some known issues with it. and we like to official only support node lts versions.

- `engines` allow node v14 and node v16+.
- `engines-next` only allows v14 and v16. So we warn on node v17+.